src/index.js

```diff
@@ -61,10 +63,40 @@ const refreshList = () => {
 
     todoContainer.append(taskDiv);
   });
+  setChks();
 };
 
 window.addEventListener('load', () => {
   refreshBtn.src = refreshIcon;
-  addBtn.src = addIcon;
+  addBt.src = addIcon;
+
+  const localStorageItem = window.localStorage.getItem('tasks');
+
+  if (localStorageItem) {
+    tasks.arrToDos = JSON.parse(localStorageItem);
+  }
+
+  refreshList();
+});
+
```

Comment:
- [ ] Please add a button to remove a single task as requested. As ofnow, I can only delete the completed tasks